src/bots/ama/index.js

```diff
@@ -0,0 +1,40 @@
+import client from '../../client';
+import { parseCommandString } from "../../utils/index";
+import { escapedBackticks } from "../../utils/embed";
+import addEvent from './subCommands/addEvent';
+import listEvents from './subCommands/listEvents';
+
+const subcommands = {'add': addEvent, 'list': listEvents}
```

Comment:
don't need to put quotes around keys, may should add an eslint rule for this sometime